src/test/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepositoryTest.java

```diff
@@ -119,17 +125,15 @@ public void findByReferenceShouldReturnCorrectSingleRecord() {
     @Test
     @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = { "classpath:sql/insert_cases.sql" })
     public void getfindByMetadataAndFieldDataReturnCorrectRecords() {
-        // Check that we have the expected test data set size, this is to ensure
-        // that state filtering is correct
-        final List<CaseDetails> resultList = template.query("SELECT * FROM case_data", this::mapCaseData);
-        assertEquals("Incorrect data initiation", 15, resultList.size());
+        assumeDataInitialised();
+
         MetaData metadata = new MetaData("TestAddressBookCase", "PROBATE");
         HashMap<String, String> searchParams = new HashMap<>();
         searchParams.put("PersonFirstName", "Janet");
         final List<CaseDetails> byMetaDataAndFieldData = caseDetailsRepository.findByMetaDataAndFieldData(metadata,
                                                                                                           searchParams);
         assertAll(
-            () -> assertThat(byMetaDataAndFieldData.size(), is(1)),
+            () -> assertThat(byMetaDataAndFieldData.size(), is(2)),
             () -> assertThat(byMetaDataAndFieldData.get(0).getId(), is(1L)),
```

Comment:
Just wonder whether you want also to assertThat byMetaDataAndFieldData.get(1) attributes

Reply:
I think that the tests are already very fragile as they are, and that would make them even more fragile. We should review our approach for test cases, it feels wrong that I had to update ~50 test cases just because I haded a new cases to the set of test cases.